app/assets/stylesheets/register.scss

```diff
@@ -0,0 +1,123 @@
+.register-page-container{
```

Comment:
replace with bootstrap classes where possible, and extract common style from login